### ISSUE TITLE
update install.md

### ISSUE THREAD

**Issue by moonshaadow** [Houston4444/RaySession#61]
Hi there !
First I want to thank you, RaySession is awesome.
In order to build, the only way working for me (ubuntu studio 18.04) is :
`make -qt`

**Comment by Houston4444** [comment]
Hi.
Thanks !
I don't know the option` make -qt`. What was the matter and how did you find this tip ?

**Comment by moonshaadow** [comment]
I was stuck with this error :
```
lrelease locale/raysession_fr_FR.ts -qm locale/raysession_fr_FR.qm
lrelease: could not exec '/usr/lib/qt5/bin/lrelease': No such file or directory
Makefile:87: recipe for target 'locale/raysession_fr_FR.qm' failed
make: *** [locale/raysession_fr_FR.qm] Error 1
```
I've tried the LRELEASE variable, but nothing better (maybe I've used it wrong).
I don't really remember how I found this trick the first time, it should be related to `qtchooser` :
```
~$ qtchooser --help
Usage:
  qtchooser { -l | -list-versions | -print-env }
  qtchooser -install [-f] [-local] <name> <path-to-qmake>
  qtchooser -run-tool=<tool name> [-qt=<Qt version>] [program arguments]
  <executable name> [-qt=<Qt version>] [program arguments]

Environment variables accepted:
 QTCHOOSER_RUNTOOL  name of the tool to be run (same as the -run-tool argument)
 QT_SELECT          version of Qt to be run (same as the -qt argument)
```
To note : on another ubuntu studio 18.04 system, with many things installed and many mess in it, `make` is enough to build RS

**Comment by moonshaadow** [comment]
Solved in #86
